mozci/task.py

```diff
@@ -616,11 +643,28 @@ def failing_tasks(self):
 
     def is_config_consistent_failure(self, minimum_count: int = 3) -> Optional[bool]:
```

Comment:
I think we should keep the `is_config_consistent_failure` logic unchanged, and make sure the "-cf" results are considered as part of the runs anyway from mozci.

Reply:
where would you recommend changing the logic for `-cf` tasks? some things to consider:
1) CF will have a higher weight to confirm failure or prove intermittent
2) will need to match CF to task with same name without `-cf` suffix

Reply:
@jmaher regarding 1, it can be a follow-up. It could go in https://github.com/mozilla/mozci/blob/d8e8c8c4437124b9e6282aece221de0b6385529f/mozci/task.py#L496 or it could go in https://github.com/mozilla/mozci/blob/d8e8c8c4437124b9e6282aece221de0b6385529f/mozci/push.py#L1121.

Regarding 2, I think we just need to update https://github.com/mozilla/mozci/blob/d8e8c8c4437124b9e6282aece221de0b6385529f/mozci/task.py#L89 so that the result is the same for a task with or without -cf.

Reply:
thanks, I can move these;  I need to make sure when stripping `-cf` that we look at what specific test/group was run, so we don't adjust counts incorrectly.

Reply:
I have a whole new approach for this, basically I have a method on tasks `is_confirmed_failure()` -> `True`, `False`, `None`

here we can add some logic here utilizing this in other commands- I have an example.  Possibly this could be optimized a bit for clarity or cpu performance.